barretenberg/cpp/src/barretenberg/stdlib/primitives/biggroup/biggroup.hpp

```diff
@@ -482,6 +482,81 @@ template <class Builder_, class Fq, class Fr, class NativeGroup> class element {
   private:
     bool_ct _is_infinity;
 
+    // Following are private methods for computation of secp256k1 wNAF. These used to be lambda
```

Comment:
I don't think you need to include the full context here in comments re them being lambda's etc - I think it's actually enough that they're under the private access specifier but if you want to be even more explicit I think its enough to say "for internal use only"